.github/workflows/tests.yml

```diff
@@ -249,4 +252,32 @@ jobs:
                 python3 -c 'import mfusepy; assert mfusepy.fuse_version_major == 2'
                 python3 -m pytest -v -rs tests
                 ;;
+              openbsd)
```

Comment:
Ideally, every commit would be green. I would move this commit to after the fixes.